apps/web-giddh/src/app/add-company/add-company.component.html

```diff
@@ -336,7 +336,7 @@
                                 </dropdown-field>
                             </div>
                             <ng-container
-                                *ngIf="secondStepForm.controls['businessType']?.value === businessTypes.Registered"
+                                *ngIf="secondStepForm.controls['businessType']?.value === businessTypes.Registered && this.selectedCountryCode !== 'US'"
```

Comment:
make a list of countries for which taxes are not applicable and check that list here

Reply:
Done